templates/scripts.tmpl/cleanup.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Please run this script only in the project root folder.
```

Comment:
Should be enforced.

Reply:
This is not the case anymore. We are copying cleanup script in every package folder separately. So I deleted the comment.